whelktool/scripts/cleanups/2022/01/lxl-3589-link-agents.groovy

```diff
@@ -94,14 +96,20 @@ Map loadAgents() {
                 agents[chip] << id
             else
                 agents[chip] = [id] as Set
+
+            chip = chip.findAll { it.key != '@type' }
```

Comment:
Maybe do `chip.remove('@type')` for clarity?

Reply:
Would prefer that too but it'll mutate `chip` so we wouldn't get two distinct map entries (one with `@type` and one without).

Reply:
Ah, of course :+1: